.travis.yml

```
@@ -32,6 +32,6 @@ script:
   # Search for PHP syntax errors.
   - find -L . -path ./vendor -prune -o -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l
   # Run PHPCS
-  - if [[ "$SNIFF" == "1" ]]; then vendor/bin/phpcs . --standard=./phpcs.xml.dist --extensions=php,lib,inc; fi
+  - if [[ "$SNIFF" == "1" ]]; then vendor/bin/php-cs-fixer fix; fi
```

```suggestion
  - if [[ "$SNIFF" == "1" ]]; then vendor/bin/php-cs-fixer fix --dry-run --diff; fi
```

cfr https://github.com/ems-project/EMSSubmissionBundle/blob/412817da1f4962ea93018af27622637706a714e3/.travis.yml#L50

Travis can only check if PHPCS is respected, it should not try to fix issues in this stage.
However you did not run the phpcs instruction as the rules did not get applied to the code!